lib/hyacinth/digital_object/type_def/json_serializable_collection.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Hyacinth
+  module DigitalObject
+    module TypeDef
+      class JsonSerializableCollection < Hyacinth::DigitalObject::TypeDef::Base
+        def initialize(type = nil)
+          super()
+          @translator = type.new if type
+          raise NotImplementedError, "Cannot instantiate #{self.class}. Instantiate a subclass instead." if self.class == JsonSerializableCollection
+        end
```

Comment:
If you add these two methods...
```
def to_serialized_form_impl(value)
          value # no conversion necessary for JsonSerializableBase objects.  JSON#generate will handle them properly.
        end

        def from_serialized_form_impl(value)
          value # no conversion necessary for JsonSerializableBase objects.  JSON#parse will generate correct form.
        end
```

Can this class just be named to `JsonSerializableBase`, and replace the existing `JsonSerializableBase` class?

Reply:
I'd like to think about that in the same scope as removing the Projects and PublishEntries typedefs - I'm unsure about it right now.